include/proxsuite/proxqp/dense/solver.hpp

```diff
@@ -1016,12 +1016,9 @@ qp_solve( //
     T rhs_pri(qpsettings.eps_abs);
     if (qpsettings.eps_rel != 0) {
       rhs_pri +=
-        qpsettings.eps_rel *
-        std::max(
-          std::max(primal_feasibility_eq_rhs_0, primal_feasibility_in_rhs_0),
-          std::max(std::max(qpwork.primal_feasibility_rhs_1_eq,
-                            qpwork.primal_feasibility_rhs_1_in_u),
-                   qpwork.primal_feasibility_rhs_1_in_l));
+        qpsettings.eps_rel * std::max(std::max(primal_feasibility_eq_rhs_0,
+                                               primal_feasibility_in_rhs_0),
+                                      qpwork.primal_feasibility_rhs_1_eq);
```

Comment:
@Bambade What is this value `primal_feasibility_rhs_1_eq`?

Reply:
it's equal to ||b||_inf. I will remove it as well for consistency.